src/Components/MailtoDropdown.js

```diff
@@ -37,19 +37,22 @@ export const MAILTO_HANDLERS = {
         countries: ['all'],
     },
     copymanually: {
-        onClick: (d) => {
-            const dismiss = () => render('', document.body, modal);
+        onClick: (d, createModal, onSuccess) => {
+            const dismiss = () => {
+                createModal(null);
+                if (onSuccess) onSuccess();
```

Comment:
There was a test for that, but I changed it (though it didn't test for it explicitly). I am not particularly invested in one solution or the other, the changed one just was easier because I was too lazy to properly manage the modal state (onSuccess might close the modal if the text modal is opened from another modal).

Reply:
I'm I hadn't noticed this in the PR, I definitely would have fallen for that and expected clicking the "Copy text manually button" to save the request. And I'm pretty sure most users (well, those who understand the "My requests" workflow, anyway…) would as well.

> There was a test for that, but I changed it (though it didn't test for it explicitly).

Implicit tests don't count. As you just proved, someone will just change the test if it fails. :P